axelor-crm/src/main/java/com/axelor/apps/crm/service/LeadService.java

```diff
@@ -69,13 +69,37 @@ public Lead convertLead(Lead lead, Partner partner, Partner contactPartner)
    */
   public boolean isThereDuplicateLead(Lead lead);
 
+  /**
+   * Set the lead status to In Process.
+   *
+   * @param lead
+   * @throws AxelorException
+   */
+  void startLead(Lead lead) throws AxelorException;
+
+  /**
+   * Set the lead status to In Process.
+   *
```

Comment:
Can you add more precision ? By just reading the javadoc, this method is the same as `startLead` or `recycleLead`

Reply:
Done